showcase/app/components/shw/flex/item.gts

```diff
@@ -4,8 +4,10 @@
  */
 
 import Component from '@glimmer/component';
-import type { ComponentLike } from '@glint/template';
+import { hash } from '@ember/helper';
+import Label from '../label';
 
+import type { ComponentLike } from '@glint/template';
```

Comment:
[question]: I don't see this used? 🤔

Reply:
It's used in the block for defining the type for the `Label` prop (L:21)

Reply:
🤦‍♂️ (it's `.gts`, these things need to be explicitly imported)